go-controller/hybrid-overlay/pkg/controller/node_linux.go

```diff
@@ -67,11 +70,58 @@ func (n *NodeController) addOrUpdatePod(pod *kapi.Pod) error {
 	if err != nil {
 		return fmt.Errorf("failed to add flows for pod %s/%s: %v", pod.Namespace, pod.Name, err)
 	}
+	subnet, err := getLocalNodeSubnet(n.nodeName)
+	if err != nil {
+		return err
+	}
+	namespace, err := n.kube.KClient.CoreV1().Namespaces().Get(pod.Namespace, metav1.GetOptions{})
```

Comment:
@rcarrillocruz we shouldn't be pulling directly from the api server here; we should be using the shared informer cache and doing a backoff if we can't find the Namespace the first time.

Reply:
@rcarrillocruz Please use the factory ObjectCache stuff here; you'll need to add a GetNamespace() call but that's trivial. And then you'll need to ExponentialBackoff() for a bit to wait for the namespace if the Pod and Namespace events show up at different times.

Reply:
@rcarrillocruz I don't think this comment was addressed?

Reply:
I may not have pushed it when pinged you, check now.